website/src/docs/aws-s3-multipart.md

```diff
@@ -107,11 +107,14 @@ The default implementation calls out to Companion’s S3 signing endpoints.
 
 ### `prepareUploadParts(file, partData)`
 
+> This option is deprecated. Use [`signPart()`][] instead.
```

Comment:
I think it might be good to keep `prepareUploadParts` as is without this `signPart` notice. For now, we are focusing on the quickfix. Later we should properly rethink this plugin and we don't want to prematurely commit to an API that we may want to change/break. 

Instead, we should put some explanation here why the API currently feels off. Such as why it's plural while we actually only do one part, in an array.